main.go

```diff
@@ -22,9 +27,28 @@ var (
 )
 
 func main() {
+	var dbHost string;
```

Comment:
A DB port would be useful as well. And As we continue to add more and more configurations I suggest to use a config file instead. In another project I read this [config file](https://github.com/kwk/docker-registry-event-collector/blob/master/config.yml) using [this code](https://github.com/kwk/docker-registry-event-collector/blob/master/config.go). By decoding a file into a config struct we can much easier pass this struct around. That is at least one benefit I see.